README.md

```diff
@@ -10,8 +10,10 @@ Welcome to the **Learning Resources Repository**! This repository is designed to
 2. [Backend Development](#-backend-development)
 3. [Mobile Development](#mobile-development)
 4. [Personal Development](#-personal-development)
-5. [Spatial Computing](#spatial-computing)
+5. [Spatial Computing](#-spatial-computing)
 6. [Machine Learning](#-machine-learning)
+7. [Technical Writing](#-technical-writing)
+
```

Comment:
No 5 and 7 links not working
For no 5  just pull from the master to get the updated version, Arafat already fixed that